text/runtime-printing.md

```diff
@@ -77,12 +77,12 @@ decl_module! {
 As you can see, our `hello-substrate` pallet has a dispatchable call that takes a single argument,
 called `origin`. The call returns a
 [`DispatchResult`](https://substrate.dev/rustdocs/v2.0.0/frame_support/dispatch/type.DispatchResult.html) which
-can be either `Ok(())` indicating that the call succeeded, or an `Err` which is demonstrated in most other recipes pallets.
+can be either `Ok(())` indicating that the call succeeded, or an `Err` which is demonstrated in most other pallets from this cookbook.
```

Comment:
Nitpick: if we are going to audit this repo for grammatical improvements, I would very much like to catch all improper usages of "which". In short, "which" should always be preceeded by a comma, as in "...or an `Err`, which is demonstrated..." Ref: https://www.1klb.com/posts/2014/01/08/that-vs-which/

Reply:
With respect to the actual change being made on this line, I'm not entirely sure if I agree with it. I may prefer something like "other Recipe pallets" - I think it is more specific than "from this cookbook".

Reply:
That works fine!